projects/fsastorefrontlib/src/cms-components/message-notification/message-notification.component.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import { AuthService, UserService } from '@spartacus/core';
+import { Observable, Subscription, combineLatest } from 'rxjs';
+import { filter, tap } from 'rxjs/operators';
+import { InboxService } from '../../core/my-account/facade/inbox.service';
+import { FSSearchConfig } from '../../core/my-account/services/inbox-data.service';
+
+@Component({
+  selector: 'cx-fs-message-notification',
+  templateUrl: './message-notification.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MessageNotificationComponent implements OnInit, OnDestroy {
+  messagesObject$: Observable<any>;
```

Comment:
Rename component as @divna-p suggested and remove from inbox folder as independent component.